sklearn/datasets/base.py

```diff
@@ -242,6 +242,108 @@ def load_files(container_path, description=None, categories=None,
                  DESCR=description)
 
 
+def load_wine(return_X_y=False):
+    """Load and return the wine dataset (classification).
+
+            .. versionadded:: 0.18
+
+    The wine dataset is a classic and very easy multi-class classification
+    dataset.
+
+    =================   ==============
+    Classes                          3
+    Samples per class               [59,71,48]
+    Samples total                  178
+    Dimensionality                  13
+    Features            real, positive
+    =================   ==============
+
+    Read more in the :ref:`User Guide <datasets>`.
+
+    Parameters
+    ----------
+    return_X_y : boolean, default=False.
+        If True, returns ``(data, target)`` instead of a Bunch object.
+        See below for more information about the `data` and `target` object.
+
+
+    Returns
+    -------
+    data : Bunch
+        Dictionary-like object, the interesting attributes are:
+        'data', the data to learn, 'target', the classification labels,
+        'target_names', the meaning of the labels, 'feature_names', the
+        meaning of the features, and 'DESCR', the
+        full description of the dataset.
+
+    (data, target) : tuple if ``return_X_y`` is True
+
+
+    The copy of UCI ML Wine Data Set dataset is
+    downloaded from:
+    https://archive.ics.uci.edu/ml/machine-learning-databases/wine/wine.data
+
+    The file has been modified:
+        -to include class labels class_0, class_1 and class_2;
```

Comment:
I don't think this will render correctly. Please check, at http://scikit-learn.org/circle?CIRCLE_BUILD_NO/modules/generated/sklearn.datasets.load_wine.html

Then again, I'm not sure this detail needs to be here as long as we're assured these changes are valid.